src/main/java/uk/ac/cam/cl/dtg/isaac/api/EventsFacade.java

```diff
@@ -486,6 +478,119 @@ public final Response getEventBookingByEventId(@Context final HttpServletRequest
+            }
+
+            List<String[]> rows = Lists.newArrayList();
+            StringWriter stringWriter = new StringWriter();
+            CSVWriter csvWriter = new CSVWriter(stringWriter);
+            StringBuilder headerBuilder = new StringBuilder();
+            headerBuilder.append(String.format("Event (%s) Attendance: Downloaded on %s \nGenerated by: %s %s \n\n",
+                    eventId, new Date(), currentUser.getGivenName(),
+                    currentUser.getFamilyName()));
+
+            List<String> headerRow = Lists.newArrayList(Arrays.asList("", ""));
+
+            rows.add(headerRow.toArray(new String[0]));
+
+            List<String> totalsRow = Lists.newArrayList();
+
+            List<String[]> resultRows = Lists.newArrayList();
+
+            for (EventBookingDTO booking : eventBookings) {
+                ArrayList<String> resultRow = Lists.newArrayList();
+                UserSummaryDTO resultUser = booking.getUserBooked();
+                RegisteredUserDTO resultRegisteredUser = this.userAccountManager.getUserDTOById(resultUser.getId());
+                String schoolId = resultRegisteredUser.getSchoolId();
+                Map<String, String> resultAdditionalInformation = booking.getAdditionalInformation();
+                BookingStatus resultBookingStatus = booking.getBookingStatus();
+                resultRow.add(resultUser.getGivenName() + " " + resultUser.getFamilyName());
+                resultRow.add(resultRegisteredUser.getRole().toString());
+                if (schoolId != null) {
+                    School school = schoolListReader.findSchoolById(schoolId);
+                    if (null != school) {
+                        resultRow.add(school.getName());
+                    } else {
+                        resultRow.add(schoolId);
+                    }
+                } else {
+                    resultRow.add(resultRegisteredUser.getSchoolOther());
+                }
+                resultRow.add(resultBookingStatus.toString());
+                resultRow.add(booking.getBookingDate().toString());
+                resultRow.add(booking.getUpdated().toString());
+                resultRow.add(resultAdditionalInformation.get("yearGroup"));
+                resultRow.add(resultAdditionalInformation.get("jobTitle"));
+                resultRow.add(resultAdditionalInformation.get("medicalRequirements"));
+                resultRow.add(resultAdditionalInformation.get("accessibilityRequirements"));
+                resultRow.add(resultAdditionalInformation.get("emergencyName"));
+                resultRow.add(resultAdditionalInformation.get("emergencyNumber"));
+                Collections.addAll(resultRows, resultRow.toArray(new String[0]));
+            }
+
+
+            rows.add(totalsRow.toArray(new String[0]));
+            rows.add(("Name,Role,School Id,Booking Status,Booking Date,Updated Date,Year Group,Job Title," +
```

Comment:
```suggestion
            rows.add(("Name,Role,School Id,Booking Status,Booking Date,Updated Date,Year Group,Job Title," +  // lgtm [java/missing-space-in-concatenation]
```
This will remove the LGTM warning, since we don't actually want an additional space here!